src/main/java/org/sonar/scm/git/blame/BlobReader.java

```diff
@@ -71,16 +71,18 @@ private RawText loadText(String path) throws IOException {
     // we use a TreeWalk to find the file, instead of simply accessing the file in the FS, so that we can use the
     // FileTreeIterator's InputStream, which filters certain characters. For example, it removes windows lines terminators
     // in files checked out on Windows.
-    FileTreeIterator it = new FileTreeIterator(repository);
-    treeWalk.reset();
-    treeWalk.addTree(it);
+    TreeWalk treeWalk = new TreeWalk(repository);
```

Comment:
treeWalk can't be cached because it's not thread safe and multiple threads call this method.